new-lamassu-admin/src/pages/Commissions/helper.js

```diff
@@ -345,12 +347,181 @@ const getOrder = ({ machine, cryptoCurrencies }) => {
   return 3
 }
 
+const createCommissions = (cryptoCode, deviceId, isDefault, config) => {
+  return {
+    minimumTx: config.minimumTx,
+    fixedFee: config.fixedFee,
+    cashOut: config.cashOut,
+    cashIn: config.cashIn,
+    machine: deviceId,
+    cryptoCurrencies: [cryptoCode],
+    default: isDefault,
+    id: v4()
+  }
+}
+
+const getCommissions = (cryptoCode, deviceId, config) => {
+  const overrides = R.prop('overrides', config)
+
+  if (overrides && !R.isEmpty(overrides)) {
```

Comment:
this can be made clearer if you invert the if. So if there's no overrides or if overrides is empty return the default commission. All of the code under the if will be moved outside =)

Reply:
Done, but i still need to keep the return default commission at the bottom, because overrides can exist, and not be applied to that pair